src/core/build/createWebpackConfig.ts

```diff
@@ -288,6 +288,25 @@ export default function createWebpackConfig({
                 },
               ],
             },
+            // Makes sure node_modules are transpiled the way we need them to be.
+            {
+              test: /\.js$/,
```

Comment:
Should we include those files coming from TS at all?

Reply:
What do you mean?

Reply:
Well, does this loader ensure that files transpiled through the `.ts` loader are also ran through the babel transformer?

Reply:
It transpiled js files in `node_modules` to the javascript version we need for our supported browsers, because a couple of modules are not in pure es5.

Reply:
What would happen if I for example, used sticky regex inside our typescript files? Would it still transform it?

Reply:
The configuration for `tsx?` files is right above this :-)